File: config/src/main/java/org/wcl/mfn/config/ui/mvc/ModelConfig.java
package org.wcl.mfn.config.ui.mvc;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.PropertySource;

@Configuration
@PropertySource("classpath:ui/mvc/model.properties")
public class ModelConfig {
    @Value("${common.view.attribute.page-title}")
    private String pageTitle;

    public String pageTitle() {
        return pageTitle;
    }
}

File: config/src/main/kotlin/org/wcl/mfn/config/ui/mvc/ModelConfig.kt
package org.wcl.mfn.config.ui.mvc

import org.springframework.beans.factory.annotation.Value
import org.springframework.context.annotation.*

@Configuration
@PropertySource("classpath:ui/mvc/model.properties")
open class ModelConfig {
    @Value("\${common.view.attribute.page-title}")
    private val pageTitle: String? = null

    fun pageTitle(): String? {
        return pageTitle
    }
}
